native/libc.c

```diff
@@ -5,8 +5,11 @@
 #include <stdarg.h>  // va_list, etc.
 #include <stdio.h>  // printf
 #include <limits.h>
+#define _XOPEN_SOURCE
```

Comment:
comment on why this is needed?

Reply:
It's mentioned in the man page for wcswidth, I'm not sure if it's actually necessary.

```
WCSWIDTH(3)                      Linux Programmer's Manual                     WCSWIDTH(3)

NAME
       wcswidth - determine columns needed for a fixed-size wide-character string

SYNOPSIS
       #define _XOPEN_SOURCE             /* See feature_test_macros(7) */
       #include <wchar.h>

       int wcswidth(const wchar_t *s, size_t n);
```

Reply:
It looks like a libc versioning thing.  I would be inclined to leave it out (since you probably get a compile time error on an ancient system), or at least add a comment that it's for wcswidth.